### PR TITLE
Edits on the IAM overview topic

### PR DESCRIPTION
**Pull request by thomascavanagh-okta**
## Description:
- **What's changed?** Edited the topic to bring up its Acrolinx score 
- **Is this PR related to a Monolith release?** No

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md](https://github.com/okta/okta-developer-docs/blob/10dd633ec9b0d99ef31d44471e663373bcaf8a2c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fiam-overview%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0197e22b-e2d9-4405-a9b1-ae17c7305ccf) | 64  | 91  | 100  | 98  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.a377a7dd-c45b-4f75-8d76-2f266459e1c5?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md](https://github.com/okta/okta-developer-docs/blob/8ba2d0b80092e8176a78e33c24db3e0b0c2f5132/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fiam-overview%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2dc71f22-8dbe-40be-9888-219222ff8b20) | 64  | 91  | 100  | 98  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.351d5618-d7e9-46d6-81bb-583db13e9f0c?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md [hunk 1]
```diff
@@ -1,50 +1,54 @@
...
+High-level designs for IAM may be called Workforce or Customer identity. The two have significant overlap in use cases and technical approaches. The most important thing is to design and build a solution that meets your particular requirements.
 
-- Workforce identity (*Workforce* or *WF*) solutions manage employee and contractor access to your organization's apps and resources. The main goal of WF solutions is to manage risk. User identities are usually assigned by the IT team. Use cases are typically administrative, such as controlling access to apps by integrating them into your single sign-on solution.
+- Workforce identity (Workforce or WF) solutions manage employee and contractor access to your organization's apps and resources. The main goal of WF solutions is to manage risk. IT teams usually assign user identities and use cases are typically administrative. For example, controlling access to apps by integrating them into your single sign-on solution.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md:45]: FYI - these for examples need to be complete sentences.
```suggestion
- Workforce identity (Workforce or WF) solutions manage employee and contractor access to your organization's apps and resources. The main goal of WF solutions is to manage risk. IT teams usually assign user identities and use cases are typically administrative. For example, you might want to control access to apps by integrating them into your single sign-on solution.
```

File: packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md [hunk 2]
```diff
@@ -1,50 +1,54 @@
...
 
-- Customer identity (*Customer Identity* and Access Management* or *CIAM*) solutions add managing customer, partner, and other external access to WF. Ease of use for customers is important, as the goal is usually to increase both customer engagement and revenue. Unlike WF users, customers commonly create their own identities, may have multiple identities, and sign in from more locations and platforms. Use cases are typically user focused, such as user experience, and building branded user registration and sign-in flows.
+- Customer identity (Customer Identity and Access Management or CIAM) solutions add the ability to manage customer, partner, and other external access to WF. Ease of use for customers is important. CIAM solutions can increase both customer engagement and revenue. Unlike WF users, CIAM users commonly create their own identities, sign in from various locations and platforms, and may have multiple identities. Use cases are typically user-focused. For example, improving the user experience (UX) of an app and building branded user registration and sign-in flows.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md:47]:
```suggestion
- Customer identity (Customer Identity and Access Management or CIAM) solutions add the ability to manage customer, partner, and other external access to WF. Ease of use for customers is important. CIAM solutions can increase both customer engagement and revenue. Unlike WF users, CIAM users commonly create their own identities, sign in from various locations and platforms, and may have multiple identities. Use cases are typically user-focused. For example, you might want to improve the user experience (UX) of an app and build branded user registration and sign-in flows.
```

File: packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md [hunk 3]
```diff
@@ -60,58 +64,59 @@ You can divide the features of an IAM system into three areas: user experience,
 
 #### Infrastructure
 
-- **Scalability:** Design your solution to allow for user growth, changing use patterns, and evolutionary changes to your applications and infrastructure, without requiring a redesign.
+- **Scalability:** Design your solution to allow for user growth and changing use patterns of your apps and infrastructure, without requiring a redesign.
 
-- **Easy integration with app stack:** Decouple your IAM solution from your applications to allow them to evolve independently and enable multiple apps to use your solution. Enable integration with your apps that optimizes performance by designing an event-based interface that is accessed using an API. Provide an SDK to make integration easier.
+- **Easy integration with app stack:** Maintain your IAM solution separately from your apps. Your apps can evolve independently and you can enable multiple apps to use your IAM solution. Enable integration with your apps that optimizes performance by designing an event-based interface that is accessed using an API. Provide an SDK to make integration easier.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md:69]:
```suggestion
- **Easy integration with app stack:** Maintain your IAM solution separately from your apps. Your apps can evolve independently, and you can enable multiple apps to use your IAM solution. Enable integration with your apps that optimizes performance by designing an event-based interface that's accessed using an API. Provide an SDK to make integration easier.
```

File: packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md [hunk 4]
```diff
@@ -60,58 +64,59 @@ You can divide the features of an IAM system into three areas: user experience,
...
 
-- **Traffic surge protection:** Provide a way to prevent sudden bursts of requests from disrupting usage or operations.
+- **Traffic surge protection:** Prevent sudden bursts of requests from disrupting usage or operations.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md:71]: avoid the use of "usage" haha - it's just jargon and very "markety"
```suggestion
- **Traffic surge protection:** Prevent sudden bursts of requests from disrupting use or operations.
```

File: packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md [hunk 5]
```diff
@@ -60,58 +64,59 @@ You can divide the features of an IAM system into three areas: user experience,
...
 
-- **High reliability** (highly redundant): Ensure that all elements of your IAM solution are highly reliable and dependable, and that every element of your solution works correctly, every time.
+- **High reliability:** Ensure that your IAM solution is dependable and that every element of your solution works correctly, every time.
 
 #### Security
 
 - **User storage and password management:** Store and manage information about your users, including their passwords, for authenticating and authorizing users.
 
-- **MFA (Multifactor Authentication):** Require a user to verify their identity in different ways. Some of these ways include knowledge, such as answers to questions, biometrics, such as a fingerprint, or possession, such as a key-card. MFA is key to controlling access to your applications.
+- **Multifactor Authentication (MFA):** Require a user to verify their identity in different ways. Some of these ways include knowledge, such as answers to questions, or biometrics, such as a fingerprint, or possession, such as a key-card. MFA is key to controlling access to your apps.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md:81]: Is there a better way that we can present this information in this sentence without using "such as" so much?
```suggestion
- **Multifactor Authentication (MFA):** Require a user to verify their identity in different ways. Some of these ways include knowledge, such as answers to questions, or biometrics, such as a fingerprint, or possession, such as a key-card. MFA is key to controlling access to your apps.
```

File: packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md [hunk 6]
```diff
@@ -60,58 +64,59 @@ You can divide the features of an IAM system into three areas: user experience,
...
 
-This is an AWS-hosted healthcare application, though you can readily adapt the abstract design to applications in any domain. The IAM design considerations are generic and could apply to any app. The Health Insurance Portability and Accountability Act (HIPAA) is the compliance requirement for healthcare, in the way that PCI is the requirement for credit card processing.
+This is an AWS-hosted healthcare app but the IAM design features are generic and can apply to apps in any domain.
 
-The application is used by the organization's employees and by other large corporations, which results in tens of millions of end users. Most end users are patients, and most primary users are physicians, nurses, and other medical personnel.
+The app is used by the organization's employees and by other large corporations, and has tens of millions of end users. Most end users are patients, and most primary users are physicians, nurses, and other medical personnel.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md:111]:
```suggestion
The app is used by the organization's employees and by other large corporations and has tens of millions of end users. Most end users are patients, and most primary users are physicians, nurses, and other medical personnel.
```

**Review by susanharper-okta — COMMENTED** [review]
couple edits and comments

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md](https://github.com/okta/okta-developer-docs/blob/60a715fdb8516b2084516977e0a979d9574b45ed/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fiam-overview%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/90bfc7ef-d05e-481f-b6d0-5f551bfad52a) | 64  | 91  | 100  | 98  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.1b1ca0fe-3bbc-43d2-9f2d-7559ba66ca20?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md](https://github.com/okta/okta-developer-docs/blob/3ea4b43fd9110a624b236a77e28bbf6d2c064588/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fiam-overview%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9ddf6240-de80-4796-879b-b62150bfaec0) | 64  | 91  | 100  | 98  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.17d63c2d-15bd-4a6a-85eb-951afadc5f26?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**